client/scripts/14araw.lua

```diff
@@ -83,23 +86,27 @@ function main(args)
 	local stayconnected = false
 	local payload = nil
 	local doconnect = true
+	local topaz_mode = false
+	local no_rats = false
 
 	-- Read the parameters
-	for o, a in getopt.getopt(args, 'corcpx:') do
+	for o, a in getopt.getopt(args, 'corcpxt3:') do
```

Comment:
You got it wrong.  
The : indicates that the param char has more bytes to read.
It should be like before,  x:  meaning payload bytes after x,   no 3: